### PR TITLE
Tests in Kotlin for `NonFungibleLocalId` and `BagOfBytes`

### PR DESCRIPTION
**Pull request by micbakos-rdx**
Tests added verifying the usage of:
1. `NonFungibleLocalId`
2. `BagOfBytes`

### CODE REVIEW

File: profile/tests/uniffi/bindings/test_bag_of_bytes.kts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    val bytes = ByteArray(byteCount)
+    return Random.nextBytes(bytes).toBagOfBytes()
+}
+
+fun ByteArray.toBagOfBytes() = newBagOfBytesFrom(bytes = this)
+
+val BagOfBytes.hex: String
+    get() = toUByteArray().toHexString()
+
+val acedBugOfBytes: BagOfBytes
```
Comment (Sajjon) [inline profile/tests/uniffi/bindings/test_bag_of_bytes.kts]: `BugOfBytes` :D :D :D intentional? :D, also present for all other extension methods below...

**Review by Sajjon — CHANGES_REQUESTED** [review]
Nice! Good job! Just change `Bug` to `Bag` in all places :)

File: profile/tests/uniffi/bindings/test_bag_of_bytes.kts (hunk 1 above)
Comment (micbakos-rdx) [reply profile/tests/uniffi/bindings/test_bag_of_bytes.kts]: Haha no way :P, will fix.

**Review by Sajjon — APPROVED** [review]
LGTM

**Pull request merged**